docs/data/data-grid/filtering-recipes/filtering-recipes.md

```diff
@@ -13,3 +13,9 @@ Currently if you want to use the [Quick filter](/x/react-data-grid/filtering/qui
 A common use case is to have certain components positioned outside of the grid. Because of the way the grid context works this might not be a straining forward thing to do. The example below illustrates how this use case can be achieved.
 
 {{"demo": "QuickFilterOutsideOfGrid.js", "bg": "inline", "defaultCodeOpen": false}}
+
+## Use non-native select in filter panel
```

Comment:
@mui/xgrid wasn't too sure about the placement into the sections. Went with this, but i am open for suggestions